portfolio/src/main/webapp/contact.html

```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+    <!-- Bootstrap CSS -->
+    <link
+      rel="stylesheet"
+      href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.0/css/bootstrap.min.css"
+      integrity="sha384-9aIt2nRpC12Uk9gS9baDl411NQApFmC26EwAOH8WgZl5MYYxFfc+NcPb1dKGj7Sk"
+      crossorigin="anonymous"
+    >
+    <!-- Bootstrap JS, Popper.js, and jQuery -->
+    <script
+        src="https://code.jquery.com/jquery-3.5.1.slim.min.js"
+        integrity="sha384-DfXdz2htPH0lsSSs5nCTpuj/zy4C+OGpamoFVy38MVBnE+IbbVYUew+OrCXaRkfj"
+        crossorigin="anonymous">
+    </script>
+    <script
+        src="https://cdn.jsdelivr.net/npm/popper.js@1.16.0/dist/umd/popper.min.js"
+        integrity="sha384-Q6E9RHvbIyZFJoft+2mJbHaEWldlvI9IOYy5n3zV9zzTtmI3UksdQRVvoxMfooAo"
+        crossorigin="anonymous">
+    </script>
+    <script
+        src="https://stackpath.bootstrapcdn.com/bootstrap/4.5.0/js/bootstrap.min.js"
+        integrity="sha384-OgVRvuATP1z7JjHLkuOU7Xw704+h835Lr+6QL9UvYjZE3Ipu6Tp75j7Bh/kR0JKI"
+        crossorigin="anonymous">
+    </script>
+    <!-- FontAwesome Icons -->
+    <script
+        src="https://kit.fontawesome.com/7cf05ff215.js"
+        crossorigin="anonymous">
+    </script>
+    <script src="script.js"></script>
+  </head>
+  <body>
+    <div id="page-container">
+      <div id="navbar">
+        <nav class="nav flex-column">
+          <a class="nav-link" href="index.html">Home</a>
+          <a class="nav-link" href="about.html">About</a>
+          <a class="nav-link" href="blog.html">Blog</a>
+          <a class="nav-link disabled" href="contact.html">Contact</a>
+          <a class="nav-link" href="gallery.html">Photo Gallery</a>
+          <a class="nav-link" href="projects.html">Projects</a>
+        </nav>
+      </div>
+      <div id="content">
+        <div id="contact-information">
+          <h1>Contact Information</h1>
+          <div id="icons-container">
+            <i
+                class="fab fa-github fa-3x icon contact-icon"
+                onclick="openNewTab('https://github.com/lyskevin')">
+            </i>
+            <i
+                class="fab fa-linkedin fa-3x icon contact-icon"
+                onclick="openNewTab('https://www.linkedin.com/in/kevin-lim-033182a1')">
+            </i>
+          </div>
+          <p>
+              Feel free to drop me an email at lyskevin@gmail.com.
```

Comment:
it is possible to add a anchor tag link to send email, in case you are interested
https://www.tutorialspoint.com/html/html_email_links.htm#:~:text=HTML%20tag%20provides%20you,mailto%20instead%20of%20using%20http.

Reply:
Added anchor tag which links to my email address. Thank you for the suggestion!